frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -144,6 +144,13 @@ class BitBox02 extends Component<Props, State> {
                         case 'attestationCheckDone':
                             this.updateAttestationCheck();
                             break;
+                        case 'keystoreAvailable':
```

Comment:
This does not seem robust. 

I don't understand the issue yet.

> The success screen did not show up after restore from backup
> due to the app.tsx rendering again and loosing the wizard state.

Why re-render, why loss of state? That does not sound like it should happen.

Reply:
Yes agree, this PR doesn't make it robust, it is just a fix by listening to an event and routing to success before other events would redirect to settings.

current issue as far as I can tell:

`app.tsx` currently listens/subscribes to `/accounts`
https://github.com/digitalbitbox/bitbox-wallet-app/blob/b67197a5b94dce6a33ef0b7771704f4678e20c46/frontends/web/src/app.tsx#L269-L276

Then checks for if props changed and if either devices or accounts are different than before it will maybeRoute
https://github.com/digitalbitbox/bitbox-wallet-app/blob/b67197a5b94dce6a33ef0b7771704f4678e20c46/frontends/web/src/app.tsx#L171-L175

Then in maybeRoute it will redirects to settings view `/device/<DEVICEID>` and jumps over the success view.

Reply:
Currently I don't like the maybeRoute method, and would prefer if each view does routing.

The only reason to forcefully route away outside of your current view/component is unplugin the device. 
But even that could be done in each component/view instead of app.tsx, and I think could be done in reusable function that can be used everywhere.

Would like to discuss this in person if you have time today.